components/global/socialLinks.tsx

```diff
@@ -9,9 +10,9 @@ const SocialLinks: StatelessComponent<SocialLinksArgs> = ({ socials }) => (
   <ul className="social">
     {socials.Twitter.Name && (
       <li>
-        <a
+        <SafeLink
           className="fa fa-twitter"
-          rel="nofollow"
+          rel="noopener nofollow"
```

Comment:
Is ``noopener`` required? it has been added to the component `<a rel="noopener" {...props}>`
```suggestion
          rel="nofollow"
```

Reply:
It will override the initial state. Not merged so we need it